### PR TITLE
Propose error output format

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- as in the title

**Related issue(s)**

- https://github.com/kyma-project/cli/issues/1992

### CODE REVIEW

File: docs/contributor/concepts/error-output-format.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+I would propose standard error format for any CLI command:
+
+```text
+kyma provision --credentials-path ~/Desktop/fs-binding.txt --plan aws --region northeurope
+
+Error:
+  failed to provision: User is unauthorized for this operation
+
+Description:
```
Comment (anoipm) [inline docs/contributor/concepts/error-output-format.md:13]: In this example we have two error descriptions called `Error` and `Description`. IMO we could rename: `Error`->`Original error` or `External system error`, `Description` -> `Error`.

File: docs/contributor/concepts/error-output-format.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Output Error Format
```
Comment (anoipm) [inline docs/contributor/concepts/error-output-format.md:1]: Good practise for ADR is putting creation date of document in the beginning.

Comment (anoipm) [inline docs/contributor/concepts/error-output-format.md:1]: ADR (architecture decision record) or ADL (architecture decision log) is well known name. Maybe we could use this name for directory `concepts`?

Comment (anoipm) [inline docs/contributor/concepts/error-output-format.md:1]: When we have many ADR's in the same directory it's easier when we put sequence number or date to file name, i.e.: `001-error-output-format.md` or `20240318-error-output-format.md`.

**Review by anoipm — DISMISSED** [review]

**Review by grego952 — DISMISSED** [review]

File: docs/contributor/ADR/001-error-output-format.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Output Error Format
+
+Creation date: 03.13.2024
```
Comment (anoipm) [inline docs/contributor/ADR/001-error-output-format.md:3]: Do we use US date format instead of international?

**Review by anoipm — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**